tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -917,6 +917,18 @@ pfcwd/test_pfc_config.py::TestPfcConfig::test_forward_action_cfg:
      conditions:
         - "asic_type in ['cisco-8000']"
 
+pfcwd/test_pfcwd_function.py:
+   xfail:
+     reason: "The test is using pfc_gen.py, which leaks out the packets."
```

Comment:
we usually used pfc_gen.py at there, why it started to fail and need  to skip recently?